### PR TITLE
Add export template for R3.1 Diagnosis

### PR DESCRIPTION
**Pull request by tstrass**
Note that the tests are not currently set up to handle R3.1, and so the test written for this template is commented out.

### CODE REVIEW

File: models/hds.go [hunk 1]
```diff
@@ -1801,6 +1801,23 @@ var hqmf_qrda_oids_r3_1 = []byte(`[
...
+  },
+  {
+    "hqmf_name": "Diagnosis",
+    "hqmf_oid": "2.16.840.1.113883.10.20.28.3.110",
+    "qrda_name": "Diagnosis",
+    "qrda_oid": "2.16.840.1.113883.10.20.24.3.135",
+    "code_displays": [
+      {
+        "code_type": "entryCode",
+        "tag_name": "code",
```
Comment (dczulada) [inline models/hds.go]: This tag_name should be "value"

File: models/hds.go [hunk 2]
```diff
@@ -1801,6 +1801,23 @@ var hqmf_qrda_oids_r3_1 = []byte(`[
...
+        "attribute": "",
+        "exclude_null_flavor": false,
+        "preferred_code_sets": ["LOINC", "SNOMED-CT", "ICD-9-CM", "ICD-10-CM"],
```
Comment (dczulada) [inline models/hds.go]: "preferred_code_sets": ["2.16.840.1.113883.3.88.12.3221.7.4"],

File: exporter/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.135.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          <targetSiteCode code="{{.EntrySection.Laterality.Code}}"
+                          codeSystem="{{oidForCodeSystem .EntrySection.Laterality.CodeSystem}}"
+                          {{if .EntrySection.Laterality.Title}}
+                            displayName="{{.EntrySection.Laterality.Title}}"
+                          {{end}}
+                          sdtc:valueSet="{{oidForCode .EntrySection.Laterality.CodedConcept (index .MapDataCriteria.FieldOids "LATERALITY")}}"
+          />
+        {{end}}
+        {{if hasAnatomicalLocation .EntrySection.AnatomicalLocation}}
+          <targetSiteCode code="{{.EntrySection.AnatomicalLocation.Code}}"
```
Comment (dczulada) [inline exporter/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.135.xml:41]: needs xsi:type="CD"

File: models/hds.go (hunk 2 above)
Comment (tstrass) [reply models/hds.go]: Why? I was just going off of [this](https://github.com/projectcypress/health-data-standards/blob/master/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.135.cat1.erb#L30).

File: exporter/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.135.xml (hunk 3 above)
Comment (tstrass) [reply exporter/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.135.xml:41]: Why? looking at the IG it seems that it's the `<value>` element that needs `xsi:type="CD"`

Comment (dczulada) [reply exporter/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.135.xml:41]: yup.  you are right, i am wrong

**Review by dczulada — APPROVED** [review]
Looks good

**Review by okeefm — APPROVED** [review]
Looks good to me.

**Comment by okeefm** [comment]
Needs to be rebased and `make` needs to be run

**Pull request merged**